tests/etl/test_transformers.py

```
@@ -0,0 +1,101 @@
+    def __init__(self, dtype: str):
+        self.dtype = dtype
+
+
+class DummySchema:
+    def __init__(self, columns):
+        self.columns = columns
+
+
+@given(df=df_strategy())
+@settings(suppress_health_check=[HealthCheck.function_scoped_fixture])
+def test_clean_and_convert_data_types(df, monkeypatch):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_clean_and_convert_data_types`

Add return type annotation: `None`

(ANN201)